### DISCUSSION TITLE
how to generate gene expression file

### DISCUSSION THREAD

**Discussion by yuzhenpeng** [hahnlab/CAGEE#33] [Q&A] [answered]
Hi, 

 I have one question. How to prepare the gene expression file contains tab-delimited columns. Could you pls give me some help.

Thanks

**Comment by benfulton** [comment, marked as answer]
The expression file simply holds levels of gene expression. It's not calculated from fasta files as is done in CAFE. You can export as tsv format from an Excel file, for example.
